components/SnippetDetails.js

```diff
@@ -1,16 +1,24 @@
 import Link from "next/link";
 import { useRouter } from "next/router";
-import useSWR from "swr";
+
 import styled from "styled-components";
+import useSWR, { mutate } from "swr";
 
-function SnippetDetails() {
+function SnippetDetails({ onDelete }) {
   const router = useRouter();
 
   const { id } = router.query;
 
   const { data, isLoading, error } = useSWR(`/api/snippets/${id}`);
+  const response = {};
+
+  if (response.ok) {
+    mutate();
+  }
```

Comment:
These lines of code are irritating, since the variable `response` is an empty object and also the SWR `mutate` function is useless here, or?